boards/arm/stm32f3_seco_d23/stm32f3_seco_d23_defconfig

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# SECO SBC-D23 board defconfig
+#
+# Copyright (c) 2022, SECO Spa
+
+CONFIG_SOC_SERIES_STM32F3X=y
+CONFIG_SOC_STM32F303XC=y
+
+# 72MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=72000000
+
+# Floating Point Options
+CONFIG_FPU=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable GPIO
+CONFIG_GPIO=y
+CONFIG_CAN_TRANSCEIVER_GPIO=y
```

Comment:
why is this needed?

Reply:
Not needed since `CONFIG_CAN_TRANSCEIVER_GPIO`: `depends on DT_HAS_CAN_TRANSCEIVER_GPIO_ENABLED`